500 call failed in blueprint YAML editor

lqueiroga opened the pull request:
- When call getLine, take line index - 1 of error to avoid undefined when it's in the latest line

Commit:
500 call failed in blueprint YAML editor

 - When call getLine, take index - 1 of error line to avoid undefined when error is in the latest line

segalaj reviewed (changes requested):
Does not work with:
```
services:
  - type: my-template
  bad
```

Commit:
Fix 500 call failed in blueprint YAML editor

- If getLine of error return undefined, create an issue on position 0 of the error line

Commit:
Refactor fix 500 call failed in blueprint YAML editor
